src/WebJobs.Script/FileTraceWriter.cs

```diff
@@ -38,9 +40,17 @@ public FileTraceWriter(string logFilePath, TraceLevel level) : base(level)
             }
             else
             {
```

Comment:
The lifetime is tied to the ScriptHost, which starts/stops regularly. Generally once we're steady state, deletions shouldn't happen very often, and when they do, I've got a try/catch to deal with any concurrent delete issues. That seemed sufficient to me. Thoughts?